src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE `taxi_db`;
+
```

Comment:
Dont forget about USE taxi_db ; without that we cant use DB and script is broken.